_episodes/l1-04-managing_contributions.md

```diff
@@ -0,0 +1,84 @@
+
+Pull requests can be created by visiting the `Pull request` tab in the repository.
+
+>#### Changing *head* and *base* branch
+>
+>By default, pull requests are based on the parent repository's default branch. You can change both the parent repository and the branch in the drop-down lists. It's important to select the correct order here; the *head branch* contains the changes you would like to make, the *base branch* is where you want the changes to be applied.
+
+>![Open a pull request]({{ site.baseurl }}/fig/create_pull_request.png "Open a pull request"){:class="img-responsive"}
+{: .callout}
+
+> ## Now you try
+> Let's revisit our recipe repository. If you have lost your copy of the recipe repository you can download a completed copy [here](https://imperialcollegelondon.github.io/introductory_grad_school_git_course/code/recipe_with_history.zip)
+> 1. Create a remote repository with your recipe repository.
+> 2. Create a new branch with some changes and push the branch to the remote repository.
+> 3. Create a pull request with a suitable title and description to merge the branch containing your changes into the main branch.
+>
+>> ## Solution
+>> 1. On GitHub.com, navigate to your repository and choose your branch which contains your changes from the "Branch" menu.
+>> ![Choose branch]({{ site.baseurl }}/fig/choose_branch.png "Choose branch"){:class="img-responsive"}
+>> 2. From the "Contribute" drop-down menu, choose the "Open pull request" button.
+>> ![Open pull request]({{ site.baseurl }}/fig/pull_request_button.png "Open pull request"){:class="img-responsive"}
+>> 3. From the *base* branch drop-down menu, choose the branch you want your changes to be merged into, and in the *compare* drop-down menu, choose the branch which contains your changes.
+>> ![Choose the base and compare branches from the drop-down]({{ site.baseurl }}/fig/base_compare_drop_down.png "Choose the base and compare branches from the drop-down"){:class="img-responsive"}
+>> 4. After giving a suitable title and description for your pull request, click the "Create pull request" button.
+>> ![Pull request title and description fields and create pull request button]({{ site.baseurl }}/fig/pr_title_description.png "Pull request title and description fields and create pull request button"){:class="img-responsive"}
+> {: .solution}
+{: .challenge}
+
+### 2. Pull request from a forked repository
+Forks are often used in large, open-source projects where you do not have write access to the upstream repository (as opposed to smaller project that you may work on with a smaller team). Proposing changes to someone else's project in this way is called the **fork and pull model**, and follows these three steps;
+1. Fork the repository.
```

Comment:
I feel this list could be more detailed. It probably does not need to be as explicit as the original - because people here has higher level - yet, they are learning, so let's walk them trough the process.

Reply:
Just below this text, there is a _challenge_ box which walks people through the process of forking.

Reply:
Oh, I see what you mean actually; the process of submitting pull requests from a fork, rather than just forking. I agree this process needs to be explained.